Dockerfile

```diff
@@ -1,28 +1,60 @@
 FROM crops/yocto:ubuntu-16.04-base
 
+LABEL description="PELUX Yocto build environment"
+LABEL maintainer="tobias.olausson@pelagicore.com"
+
+# Enables us to overwrite the user ID for the yoctouser. See below
 ARG userid=1000
 
 USER root
 
-RUN mkdir /var/run/sshd
-
 # Install dependencies in one command to avoid potential use of previous cache
 # like explained here: https://stackoverflow.com/a/37727984
-RUN apt-get update \
-    && apt-get install -y openssh-server inetutils-ping iptables cvs subversion coreutils python3-pip libfdt1 python-pysqlite2 help2man libxml2-utils libsdl1.2-dev graphviz qemu-user g++-multilib curl
+RUN apt-get update && apt-get install -y \
+        openssh-server \
+        inetutils-ping \
+        iptables \
+        cvs \
+        subversion \
+        coreutils \
+        python3-pip \
+        libfdt1 \
+        python-pysqlite2 \
+        help2man \
+        libxml2-utils \
+        libsdl1.2-dev \
+        graphviz \
+        qemu-user \
+        g++-multilib \
+        curl
```

Comment:
Add a \ at the end of this row as well, then you can mix and match easier

Reply:
Adding a backslash there makes the Dockerfile unparseable, so no.

Reply:
Or rather, it gives a warning that will become an error in future versions.

Reply:
:/